README.md

```diff
@@ -36,8 +36,9 @@ to Mac users, but Lima can be used for non-container applications as well.
 
 ## Adopters
 
-- [Rancher Desktop](https://rancherdesktop.io/): Kubernetes and container management to the desktop
 - [Colima](https://github.com/abiosoft/colima): Docker (and Kubernetes) on macOS with minimal setup
+- [Lima xbar plugin](https://github.com/unixorn/lima-xbar-plugin) - [xbar](https://xbarapp.com/) plugin to start/stop VMs from the menu bar and see their running status.
+- [Rancher Desktop](https://rancherdesktop.io/): Kubernetes and container management to the desktop
```

Comment:
I don’t think the list has to be alphabetically sorted.

Also, could you squash commits?

Reply:
> I don’t think the list has to be alphabetically sorted.

You'll need to let @unixorn know which order you prefer. Is it by original release date or something else?

> Also, could you squash commits?

Yes, the change done via the Github website is also missing the DCO line.

Reply:
Squashed the commits and signed the new squashed one.

I'm happy to revert the sort to the original order. It doesn't have to be alphabetical, I just feel that makes it easier to find things - not an issue now with three items, but for the future.